controllers/mysqlcluster_controller.go

```diff
@@ -968,6 +968,9 @@ func (r *MySQLClusterReconciler) reconcileV1StatefulSet(ctx context.Context, req
 		if needRecreate {
 			metrics.StatefulSetRecreateErrorTotal.WithLabelValues(cluster.Name, cluster.Namespace).Inc()
 		}
+		if apierrors.IsInvalid(err) {
```

Comment:
I was going to write a test, but couldn't figure out how to cause this with the current MySQLCluster spec.
Is there a better way?

Reply:
@d-kuro 
Sorry, I misunderstood the specification of the MySQLCluster.
Certainly, the invalid update will not happen...

May I close this PR?